.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "bracketSpacing": false
+}
```

Comment:
Please remove this file.

The project uses ESLint, configured in `package.json`: https://github.com/openlayers/jsdoc-plugin-typescript/blob/80089c91ccf971b26f056ec570669b99a7048b87/package.json#L35-L37

Reply:
Seem to be some formatting changes on `main` that aren't correct. Is `eslint index.js --fix` the command I should run?